site/content/integrate/apps/system-admin-guide/_index.md

```diff
@@ -0,0 +1,6 @@
+---
+title: "System Admin Guide"
+heading: "System Admin Guide"
+description: "System Admin Guide for Mattermost Apps."
+weight: 90
```

Comment:
Is it worth adding this right now?

Reply:
No, it's definitely not. I struggled with figuring out the structure so I'll remove it for now.